packages/create-react-native-library/src/utils/packageManager.ts

```diff
@@ -0,0 +1,8 @@
+import fs from 'fs-extra';
+import path from 'path';
+
+export async function determinePackageManager() {
```

Comment:
Is `bun` supported?

Reply:
We don't officially support `bun` right now. That being said, there is no real reason libraries bob generates shouldn't work with `bun`. It might require some changes tho.